.github/workflows/ci.yaml

```diff
@@ -0,0 +1,56 @@
+name: CI
+
+on: [ push, pull_request ]
+
+jobs:
+  package-lock-lint:
+    name: package-lock lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - name: lint package-lock
+        run: npx lockfile-lint --path package-lock.json --allowed-hosts npm --validate-https
+
+  test:
+    strategy:
+      matrix:
+        platform: [ubuntu-latest]
+        node: ['8', '10', '12']
```

Comment:
You can upgrade the node versions to 10, 12 and 14

Reply:
doesn't it require a major version update?

Reply:
Right, saw the readme... So just add `14`, without removing the rest.